libraries/helpers.rb

```diff
@@ -581,3 +581,25 @@ def execute_command(command, user = "root", timeout = 300, raise_on_error = true
   raise_command_error(command, cmd) if raise_on_error && cmd.error?
   cmd.stdout.strip
 end
+
+#
+# Disable service
+#
+def disable_service(service, platform_families = %i(rhel amazon debian), operations = :disable)
+  if platform_family?(platform_families)
+    service service do
+      action operations
+    end
+  end
+end
+
+#
+# Check if a service is disabled
+#
+def service_is_disabled(service, platform_families = %i(rhel amazon debian))
+  if platform_family?(platform_families)
+    execute "check #{service} service is disabled" do
+      command "systemctl is-enabled #{service} && exit 1 || exit 0"
+    end
+  end
+end
```

Comment:
can we move disable_service in an helpers file for the install cookbook and service_is_disabled in an helpers file for the test cookbook?

Reply:
minor: I'd rename to is_service_disabled

Reply:
Done